app.rb

```diff
@@ -54,6 +54,19 @@
   }
 }
 
+FLUENT_PACKAGE_VERSIONS = {
+  :v5 => {
+    linux: "5.0.1 (General Availability)",
+    mac: "5.0.1 (General Availability)",
+    win: "5.0.1 (General Availability)"
+  },
+  :v5_lts => {
+    linux: "5.0.1 (General Availability)",
+    mac: "5.0.1 (General Availability)",
+    win: "5.0.1 (General Availability)"
+  },
+}
```

Comment:
Do we need `(General Availability)` supplement?

Reply:
It may be redundant, removed.